frontend/src/modules/GraphLibrary/components/GraphStatus/components/MeasurementElement/MeasurementElement.module.scss

```diff
@@ -15,11 +16,11 @@
 
   &.nodeSelected {
     border-color: var(--primary-color);
-    background: var(--bg-color-selected);
+    background-color: #28292F;
```

Comment:
Why are the colors now hardcoded? I would expect that the colors are defined in a global file and referenced from the css files